scripts/setup_jasmine.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Why not add jasmine as a developer dependency in `package.json` and add a `postinstall` script to move it?

Reply:
Because the whole point of this is that we don't want to run Jasmine through npm, but in the browser. And jasmine in the browser needs the `lib` folder and the `specrunner.html` to work.
Also, jasmine is not the only thing that we need to setup, so it is more convenient to have everything that is done only once to setup the repo in the same setup file. You are supposed to run this file once and then forget about it. It's there just to make the dev's lives easier.

Reply:
Makes sense.